diff --git a/node_modules/react-native/scripts/find-node-for-xcode.sh b/node_modules/react-native/scripts/find-node-for-xcode.sh
index e928cd8..4f976e4 100644
--- a/node_modules/react-native/scripts/find-node-for-xcode.sh
+++ b/node_modules/react-native/scripts/find-node-for-xcode.sh
@@ -20,6 +20,7 @@ set -e
 # remove global prefix if it's already set
 # the running shell process will choose a node binary and a global package directory breaks version managers
 unset PREFIX
+unset npm_config_prefix

 # Support Homebrew on M1
 HOMEBREW_M1_BIN=/opt/homebrew/bin
