diff --git a/node_modules/react-native/React/Views/RCTModalHostView.m b/node_modules/react-native/React/Views/RCTModalHostView.m
index 65428a0..b64f914 100644
--- a/node_modules/react-native/React/Views/RCTModalHostView.m
+++ b/node_modules/react-native/React/Views/RCTModalHostView.m
@@ -116,10 +116,12 @@ - (void)didUpdateReactSubviews

 - (void)dismissModalViewController
 {
-  if (_isPresented) {
-    [_delegate dismissModalHostView:self withViewController:_modalViewController animated:[self hasAnimationType]];
-    _isPresented = NO;
-  }
+  dispatch_async(dispatch_get_main_queue(), ^{
+    if (self->_isPresented) {
+      [self->_delegate dismissModalHostView:self withViewController:self->_modalViewController animated:[self hasAnimationType]];
+      self->_isPresented = NO;
+    }
+  });
 }

 - (void)didMoveToWindow
diff --git a/node_modules/react-native/scripts/find-node.sh b/node_modules/react-native/scripts/find-node.sh
index 11338d0..7fb3e44 100755
--- a/node_modules/react-native/scripts/find-node.sh
+++ b/node_modules/react-native/scripts/find-node.sh
@@ -9,6 +9,7 @@ set -e
 # remove global prefix if it's already set
 # the running shell process will choose a node binary and a global package directory breaks version managers
 unset PREFIX
+unset npm_config_prefix

 # Support Homebrew on M1
 HOMEBREW_M1_BIN=/opt/homebrew/bin
